packages/apps/transfer/src/services/faucet/index.ts

```diff
@@ -28,11 +28,11 @@ const debug = Debug('kadena-transfer:services:faucet');
 export const fundExistingAccount = async (
   account: string,
   chainId: ChainwebChainId,
+  amount = 100,
   onPoll?: (
     transaction: IPactCommand & ICommandBuilder<Record<string, unknown>>,
     pollRequest: Promise<IPollResponse>,
   ) => void,
-  amount = 100,
```

Comment:
don't we need this?

Reply:
It's still in there, just move one position up of parameters, see change on the right